ipld/resolver/src/service.rs

```diff
@@ -691,7 +691,7 @@ async fn download_blob(iroh: Iroh, hash: Hash, node_addr: NodeAddr) -> anyhow::R
 
     // Delete the temporary tag (this might fail as not all nodes will have one).
     // TODO: this needs to be tagged with a "user id"
-    let tag = iroh::blobs::Tag(format!("temp-{hash}").into());
+    let tag = iroh::blobs::Tag(format!("temp-seq-{hash}").into());
```

Comment:
nit: should `"stored-{hash}"` above be `"stored-seq-{hash}"` to match the new pattern?

Reply:
in which case, https://github.com/recallnet/ipc/blob/main/recall/syscalls/src/lib.rs#L47 would have to change as well

Reply:
fixed